core/src/test/java/fr/sncf/osrd/envelope/MRSPEnvelopeBuilderTest.java

```diff
@@ -140,13 +140,14 @@ public void speedLimitByCategory() {
         for (var dir : Direction.values())
             speedSections.put(dir, map);
         setTrackSpeedSections(edge, speedSections);
-        var path = List.of(
+        List<TrackRangeView> path = List.of(
                 new TrackRangeView(20, 100, new DiTrackEdgeImpl(edge, Direction.FORWARD))
         );
 
-        var mrspNoTags = MRSP.from(path, REALISTIC_FAST_TRAIN, false, List.of());
-        var mrspFastShort = MRSP.from(path, REALISTIC_FAST_TRAIN, false, List.of("FAST", "SHORT"));
-        var mrspFastLong = MRSP.from(path, REALISTIC_FAST_TRAIN, false, List.of("FAST", "LONG"));
+        var mrspNoTags = MRSP.from(path, REALISTIC_FAST_TRAIN, false, null);
+        var mrspFast = MRSP.from(path, REALISTIC_FAST_TRAIN, false, "FAST");
+        var mrspShort = MRSP.from(path, REALISTIC_FAST_TRAIN, false, "SHORT");
+        var mrspLong = MRSP.from(path, REALISTIC_FAST_TRAIN, false, "LONG");
```

Comment:
Yeah I didn't have the heart to delete them

Do these make sense ?

Reply:
looks like it